testing/tests/DevExpress.ui.widgets.scheduler/recurrenceEditor.tests.js

```diff
@@ -478,6 +479,25 @@ QUnit.test('Recurrence editor should process values from repeat-on-editor correc
     assert.equal(this.instance.option('value'), 'FREQ=WEEKLY;BYDAY=SU,MO,TU');
 });
 
+const BUTTON_CLASS = 'dx-button';
```

Comment:
Should we use this const out of the test?
I suggest use the `dx-button` class directly in `buttonSelector`.